PowerShell/CipherTrustManager/Tokens/CipherTrustManager-Tokens.psm1

```diff
@@ -27,8 +27,8 @@ public enum CM_TokensGrantTypes {
 $target_uri = "/auth/tokens"
 $target_revoke_uri = "/auth/revoke"
 $target_selfdomain_uri = "/auth/self/domains"
-$target_authkey_uri = "/auth/auth-key"
-$target_authkey_rotation_uri = "/auth/rotate-auth-key"
+#$target_authkey_uri = "/auth/auth-key"
```

Comment:
if we are not gonna need them, can we just remove them?

Reply:
I guess. I didn't add those initially, they were already there. I can double check, but yeah... if we don't need them in the module, we could drop them.